data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,325 @@
+			`	It then looks you directly in the eyes. "Remember what I have told you, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
+				accept
+	on enter Dom'us
+		"reputation: Hicemus" += 10
+		conversation
+			`Entering the system with the translator, you prepare to broadcast your first message.`
+			choice
+				`	"Hello. I am Captain <first> <last>, here to make contact."`
+				`	"Greetings, strangers. I seek to help you."`
+				`	"I come in peace."`
+			`	You notice that your translator starts to flash a sequence of lights, and you point it towards your ship's camera. For a minute, nothing happens at all.`
+			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever these aliens are saying to you.`
+			`	"Welcome world-traveler. We greet you in our home-space."`
+			`	The colors stop before rushing over your screen again.`
+			`	"What is your intention-goal here?"`
+			choice
+				`	"I am here to make peaceful contact with you."`
+				`	"I want to help one of your stranded ships."`
+			`	Again a long pause. "Your intention-goals are welcome. Please dock to capital-station."`
+			`	Along the message comes a small map of the station with the docking ports highlighted.`
+	on complete
+		conversation
+			`Thanks to the map you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you ever saw before. There are no visible pathways, but instead foggy hallways, some of which even ascend vertically. A crowd of beings gathered in front of your airlock each flash a multitude of colors. A group of five floats in front of the crowd, and the middle one starts pulsating in calm tones.`
+			`	"Welcome, outer-being, to our capital. We are the Incipias."`
+			`	It stops as if awaiting an answer.`
+			choice
+				`	"Greetings, Incipias. I am a human."`
+				`	"I thank you for your hospitality."`
+			`	Your translator begins rapidly flashing. The Incipias seem to respond to this - movement and colors begin to diffuse through the crowd like waves.`
+			`	The 'speaker' continues, "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
+			choice
+				`	"No, there are others too."`
+				`	"The galaxy is full of other beings."`
+			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we help you?"`
+			choice
+				`	(Tell them about the stranded ship and how I want to help them.)`
```

Comment:
Should this, and the one earlier, be went?

Reply:
no you only provided some assistance, they still need help to return

Reply:
You can mark it as resolved.